cardinal/server/handler/debug.go

```diff
@@ -0,0 +1,66 @@
+package handler
+
+import (
+	"encoding/json"
+
+	"github.com/gofiber/fiber/v2"
+
+	"pkg.world.dev/world-engine/cardinal/search/filter"
+	servertypes "pkg.world.dev/world-engine/cardinal/server/types"
+	"pkg.world.dev/world-engine/cardinal/types"
+)
+
+type DebugStateRequest struct{}
+
+type debugStateElement struct {
+	ID         types.EntityID             `json:"id"`
+	Components map[string]json.RawMessage `json:"components" swaggertype:"object"`
+}
+
+type DebugStateResponse []*debugStateElement
+
+// GetDebugState godoc
+//
+//	@Summary		Get information on all entities and components in world-engine
+//	@Description	Displays the entire game state.
+//	@Produce		application/json
+//	@Success		200	{object}	DebugStateResponse
```

Comment:
nit: we should probably annotate possible internal server error (Error 500)

Reply:
done